src/test/java/org/folio/rest/api/PrecedingSucceedingTitleTest.java

```diff
@@ -0,0 +1,315 @@
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class PrecedingSucceedingTitleTest extends ResourceTestBase {
+
+  private final static String INSTANCE_TYPE_ID_TEXT = "6312d172-f0cf-40f6-b27d-9fa8feaf332f";
+  private final static String HRID = "inst000000000022";
+  private final static String TITLE = "A web primer";
+
+  @BeforeClass
+  public static void beforeAll() {
+    StorageTestSuite.deleteAll(itemsStorageUrl(""));
+    StorageTestSuite.deleteAll(holdingsStorageUrl(""));
+    StorageTestSuite.deleteAll(instancesStorageUrl(""));
+  }
+
+  @Test
+  public void testPrecedingSucceedingTitleBasicCrud() throws MalformedURLException,
+    InterruptedException, TimeoutException, ExecutionException {
+
+    String entityPath = precedingSucceedingTitleUrl("").getPath();
+    JsonObject instance1Response = createInstance("Title One", INSTANCE_TYPE_ID_TEXT);
+    JsonObject instance2Response = createInstance("Title Two", INSTANCE_TYPE_ID_TEXT);
+    JsonArray identifiers = new JsonArray();
+    identifiers.add(identifier(UUID_ISBN, "9781473619777"));
+
+    PrecedingSucceedingTitle precedingSucceedingTitle = new PrecedingSucceedingTitle(
+      instance1Response.getString(Instance.ID_KEY), instance2Response.getString(Instance.ID_KEY),
+      TITLE, HRID, identifiers);
+    Response postResponse = createReferenceRecord(entityPath, precedingSucceedingTitle);
+    assertThat(postResponse.getStatusCode(), is(HttpURLConnection.HTTP_CREATED));
+
+    String entityUUID = postResponse.getJson().getString("id");
+
+    String updateProperty = PrecedingSucceedingTitle.TITLE_KEY;
+
+    testGetPutDeletePost(entityPath, entityUUID, precedingSucceedingTitle, updateProperty);
+  }
+
+  @Test
+  public void canCreateConnectedPrecedingSucceedingTitle() throws MalformedURLException,
+    InterruptedException, ExecutionException, TimeoutException {
+
+    JsonObject instance1Response = createInstance("Title One", INSTANCE_TYPE_ID_TEXT);
+    JsonObject instance2Response = createInstance("Title Two", INSTANCE_TYPE_ID_TEXT);
+    String instance1Id = instance1Response.getString(Instance.ID_KEY);
+    String instance2Id = instance2Response.getString(Instance.ID_KEY);
+
+    JsonArray identifiers = new JsonArray();
+    identifiers.add(identifier(UUID_ISBN, "9781473619777"));
+
+    PrecedingSucceedingTitle precedingSucceedingTitle = new PrecedingSucceedingTitle(
+      instance1Id, instance2Id, TITLE, HRID, identifiers);
+
+    Response response = createReferenceRecord(
+      precedingSucceedingTitleUrl("").getPath(), precedingSucceedingTitle);
+    assertThat(response.getStatusCode(), is(HttpURLConnection.HTTP_CREATED));
+
+    assertPrecedingSucceedingTitleIsCreated(response, instance1Id, instance2Id, TITLE, HRID, identifiers);
+    deletePrecedingSucceedingTitle(response.getJson().getString("id"));
```

Comment:
You can use `org.folio.rest.api.StorageTestSuite#deleteAll(tenantId, tableName)` to remove entire table in the `@BeforeClass` method instead.